README.md

```diff
@@ -7,916 +7,62 @@
 
-  - [General model](#general-model)
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- Service management with [layers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)
+- Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/)
+- Identities
+- Can be used in Machines and containers
+- [CLI](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/cli-commands/), [HTTP API and Go client](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/)
 
-Pebble is organized as a single binary that works as a daemon and also as a
-client to itself. When the daemon runs it loads its own configuration from the
-`$PEBBLE` directory, as defined in the environment, and also records in
-that same directory its state and unix sockets for communication. If that variable
-is not defined, Pebble will attempt to look for its configuration from a default
-system-level setup at `/var/lib/pebble/default`. Using that directory is encouraged
-for whole-system setup such as when using Pebble to control services in a container.
+## Quick start
 
-The `$PEBBLE` directory must contain a `layers/` subdirectory that holds a stack of
-configuration files with names similar to `001-base-layer.yaml`, where the digits define
-the order of the layer and the following label uniquely identifies it. Each
-layer in the stack sits above the former one, and has the chance to improve or
-redefine the service configuration as desired.
+Prerequisites:
 
-## Layer configuration examples
+- A Linux machine.
+- Python 3.x (used to run a basic HTTP server as a sample service managed by Pebble).
 
-Below is an example of the current configuration format.
-For full details of all fields, see the [complete layer specification](#layer-specification).
-
-```yaml
+```bash
+git clone https://github.com/canonical/pebble.git
+cd pebble
```

Comment:
For a quick start that's for users not Pebble devs, probably it would be cleaner to `go install github.com/canonical/pebble/cmd/pebble@latest` or `sudo snap install pebble` and also adjust the command for running pebble, right? Probably the snap would be the best way, so that you don't have to knowingly be doing anything with Go.

Reply:
Agreed the either of these is better for normal users than cloning. We also present a different way again in the tutorial (download the pre-compiled binaries and unpack). The snap approach looks tantalisingly simple ... but maybe for the README the `go install ...@latest` is best.